src/pages/accounts/SignUpPage.tsx

```diff
@@ -12,46 +16,35 @@ const SignUpPageContainer = styled.div`
 `;
 
 const SignUpContents = styled.div`
-	width: 250px;
+	max-width: 300px;
```

Comment:
Is this supposed to be using px? Or should it be using rem or em?

Reply:
So this I actually don't want to vary with font size (which rem and em do). Since this is a top level layout px is fine. And it is meant to be based on a users screen in px.